tests/test_jdatetime.py

```diff
@@ -385,6 +386,8 @@ def test_strptime_handle_b_B_directive(self):
             with self.subTest(date_string=date_string, date_format=date_format):
                 date = jdatetime.datetime.strptime(date_string, date_format)
                 self.assertEqual(jdatetime.datetime(1400, 2, 14), date)
+        two_chars_month = jdatetime.datetime.strptime('14 دی 1400', '%d %B %Y')
```

Comment:
hmmm, seems all the tests listed above are for `14 Ordibehesht 1400`. Please add a new test and move it to the new one. something like `test_strptime_handle_b_B_directive_two_chars_month`

Reply:
yes, it makes sense now in the separate test method.